flink-streaming-connectors/flink-connector-kafka-0.10/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaProducer010.java

```diff
@@ -102,11 +102,11 @@
 	 * @param serializationSchema User defined (keyless) serialization schema.
 	 * @param producerConfig Properties with the producer configuration.
 	 */
-	public static <T> FlinkKafkaProducer010Configuration writeToKafka(DataStream<T> inStream,
-										String topicId,
-										SerializationSchema<T> serializationSchema,
-										Properties producerConfig) {
-		return writeToKafka(inStream, topicId, new KeyedSerializationSchemaWrapper<>(serializationSchema), producerConfig, new FixedPartitioner<T>());
+	public static <T> FlinkKafkaProducer010Configuration writeToKafkaWithTimestamps(DataStream<T> inStream,
```

Comment:
Add the generic type parameter `T` to `FlinkKafkaProducer010Configuration` here too?